packages/suite-web/e2e/tests/metadata/wallet-metadata.test.ts

```diff
@@ -110,7 +110,7 @@ describe('Metadata - wallet labeling', () => {
             cy.getTestElement('@switch-device/wallet-on-index/1/toggle-remember-switch').click({
                 force: true,
             });
-            cy.wait(200); // wait for data to save to persistent storage. currently this is not reflected in UI
+            cy.wait(1000); // wait for data to save to persistent storage. currently this is not reflected in UI
```

Comment:
unfortunatelly this could be the solution. can't we find something bettern than waiting? query indexxed db for some data, or for its status from the test directly? wait for some "ready" event?